docs/use_cases/model_applications/s2s_mid_lat/UserScript_obsERA_obsOnly_Blocking.py

```diff
@@ -1,173 +1,209 @@
 """
-Blocking Calculation: ERA RegridDataPlane, PcpCombine, and Blocking python code
-================================================================================
+UserScript: Calculate Blocking for the ERA and possibly RegridDataPlane and PcpCombine
+======================================================================================
 
 model_applications/
```

Comment:
```suggestion
model_applications/s2s_mid_lat/UserScript_obsERA_obsOnly_Blocking.py
```
The [template](https://github.com/dtcenter/METplus/blob/develop/docs/use_cases/use_case_documentation_template.py) has the full path on one line.